bindings/wasm/src/crypto/wasm_signature_options.rs

```diff
@@ -37,12 +37,6 @@ impl From<SignatureOptions> for WasmSignatureOptions {
   }
 }
 
-impl From<WasmSignatureOptions> for SignatureOptions {
-  fn from(options: WasmSignatureOptions) -> Self {
-    options.0
-  }
-}
```

Comment:
I don't think this implementation should be removed entirely, the previous comment was about overwriting it with an implementation for a borrowed parameter.

We use `From` for clarity ,compared to accessing the inner field directly with `x.0` everywhere, and with functional-style calls like `map`.

Reply:
I added the implementation for the borrowed parameter.